recipes/nannos/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "nannos" %}
+{% set version = "2.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/{{ name }}/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: e1190eef877346c7fa0f3e87ea6c3de6af4bc90f1667ca56611e6b6b1d3b6181
+
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  skip: true  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy >=1.9
+    - scipy >=1.0
+    - autograd >=1.4
+    - matplotlib-base
+    - shapely
+    - joblib
+    - colorlog
+    - psutil
+    - ipython
+    - nlopt
+    - pytorch       # [not win]
+    - pytorch-cpu   # [cuda_compiler_version == "None" and not win]
+    - pytorch-gpu   # [cuda_compiler_version != "None" and not win]
+    - magma         # [not osx and not win]
+    - jax           # [not win]
+    - jaxlib        # [not win]
+    - pyvista
+    - pythreejs
+    - ipyvtklink
+    - panel
+    - importlib_metadata   # [py<38]
+
+test:
+  imports:
+    - {{ name }}
+  source_files:
+    - test/conda_test.py
+  commands:
+    - pytest -svv ./test/conda_test.py
+  requires:
+    - pytest
+
+about:
+  home: https://gitlab.com/{{ name }}/{{ name }}
```

We usually prefer "cliackable" links in the recipe so we can follow along and check the metadata.

```suggestion
  home: https://gitlab.com/nannos/nannos
```